usbl_seatrac.orogen

```diff
@@ -54,8 +54,12 @@ task_context "Task", subclasses: "iodrivers_base::Task" do
     # Maximum time that a beacon is not communicated with before its position filter
     # is reset
     property "xcvr_posflt_tmo", "base/Time"
+    # Ping's minimum refresh_period
+    property "ping_refresh_period", "base/Time"
```

Comment:
I guess ping_update_rate  is a better name

Reply:
its not the rate, its the period. ping_update_period?